models/stg_sap.yml

```diff
@@ -0,0 +1,2158 @@
+            - endda
+            - begda
+            - seqnr
+    columns:
+      - name: mandt
+        description: Client
+      - name: pernr
+        description: Personnel number
+      - name: subty
+        description: Subtype
+      - name: objps
+        description: Object information
+      - name: sprps
+        description: Lock indicator for HR master data record
+      - name: begda
+        description: Start date
+      - name: endda
+        description: End date
+      - name: seqnr
+        description: Number of infotype record with same key
+      - name: aedtm
+        description: Changed on
+      - name: flag1
+        description: Reserved field/unused field
+      - name: flag2
+        description: Reserved field/unused field 
+      - name: flag3
+        description: Reserved field/unused field 
+      - name: flag4
+        description: Reserved field/unused field 
+      - name: grpvl
+        description: Grouping value for personnel assignments
+      - name: histo
+        description: Historical record flag
+      - name: itbld
+        description: Infotype screen control
+      - name: itxex
+        description: Text exists for infotype
+      - name: ordex
+        description: Confirmation fields exist
+      - name: preas
+        description: Reason for changing master data
+      - name: refex
+        description: Reference fields exist
+      - name: rese1
+        description: Reserved field/unused field of length 2
+      - name: rese2
+        description: Reserved field/unused field of length 2 
+      - name: rfp01
+        description: Reference personnel number
+      - name: rfp02
+        description: Reference personnel number 
+      - name: rfp03
+        description: Reference personnel number 
+      - name: rfp04
+        description: Reference personnel number 
+      - name: rfp05
+        description: Reference personnel number 
+      - name: rfp06
+        description: Reference personnel number 
+      - name: rfp07
+        description: Reference personnel number 
+      - name: rfp08
+        description: Reference personnel number 
+      - name: rfp09
+        description: Reference personnel number 
+      - name: rfp10
+        description: Reference personnel number
+      - name: rfp11
+        description: Reference personnel number 
+      - name: rfp12
+        description: Reference personnel number
+      - name: rfp13
+        description: Reference personnel number
+      - name: rfp14
+        description: Reference personnel number
+      - name: rfp15
+        description: Reference personnel number
+      - name: rfp16
+        description: Reference personnel number
+      - name: rfp17
+        description: Reference personnel number
+      - name: rfp18
+        description: Reference personnel number 
+      - name: rfp19
+        description: Reference personnel number 
+      - name: rfp20
+        description: Reference personnel number
```

Comment:
Are these all intended to be the same description?

Reply:
Yep!
<img width="650" alt="Screenshot 2023-07-06 at 4 32 48 AM" src="https://github.com/fivetran/dbt_sap_source/assets/108772760/29d76698-a532-431d-9d43-c20ad0080207">